recyclerview/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
No need for this .gitignore file. This is already taken care of in the outer .gitignore file